EventKit/eventkit-cloud
# Send rocketchat notifications when jobs fail early

## Description of Improvement
Jobs failing in the pre-run stage were not sending notifications via rocketchat, this sends notifications from `pick_up_run_task`

## How to test
Configure rocketchat notifications. Test by requesting a new data product from the coverages tab. 
Recreating the error organically may be challenging. Instead, raise an exception in the `try` block of pick_up_run_task. 
Run a job, verify that a notification is sent to rocketchat when the job fails. 

## Quality Assurance 

The following items have been updated:
 
## Screenshots



:smile:
